### DISCUSSION TITLE
Sunshine not detecting my Nvidia RTX 4070 Ti supports AV1 codec

### DISCUSSION THREAD

**Discussion by aBigJerk** [LizardByte/community#579] [Sunshine]
### Select Topic Area

Question

### Body

Like the title says, Sunshine will not give me the option to use the **AV1 codec** with my **RTX 4070 Ti** gpu that 100% does support it. 

No matter which option I choose within the dropdown under **AV1 Support**, the **AV1** option is missing from the dropdown under **Force Encoder**.

Any ideas?

**Comment by ReenigneArcher** [comment]
Does the client support av1 decoding?

**Reply by aBigJerk** [reply]
It sure does.

**Reply by aBigJerk** [reply]
I see what the issue is, but I don't have a clue as to the why. On top of that I don't even know where to start in an attempt to fix it. See the snip I made? Yeah, I don't have a Radeon RX 580. I don't even have a Radeon anything. My gpu is a RTX 4070 Ti

Why is Sunshine not detecting the correct gpu in my rig? How and where does it get this information from?

![image](https://github.com/user-attachments/assets/5d08887b-29fb-482e-95db-c996c2697e2e)

**Reply by ReenigneArcher** [reply]
That's just a placeholder value for the UI... Sunshine isn't using that.

**Reply by aBigJerk** [reply]
DAMNIT! I thought I was onto something. Boy, do I feel like a young newb all over again. Feels not great.

Thank you for setting me straight. You're right. When I checked the log, Sunshine identifies my gpu brand and model correctly. I don't see anything else in there that seemed sus.

So what gives? I'm not the only one experiencing this. I've seen a couple of peeps inquiring about the same issue, but sadly there are no answers or info on this particular subject whatsoever. Heavy sigh.

I'm boosted to see how AV1 performs with Sunshine and how it compares to using HEVC (which works well).

I get great results using AV1 for my live streaming DJ sessions. So...I'm a fan.

[a comment by an automated account is not shown]
